extended homepage

tusharkandpal opened the pull request:
- moved iframe styles to global.css
- quick introduction
- Wings UI setup link
- font-awesome link

tusharkandpal committed:
extended homepage

- moved iframe styles to global.css
- quick introduction
- Wings UI setup link
- font-awesome link

Priya-Patel31 commented on utilities/global.css:
```diff
@@ -25,6 +25,19 @@ a {
+  min-height: 50vh;
+}
+iframe {
+  border: 0;
```
use border:none instead of border: 0

Priya-Patel31 commented on utilities/global.css:
```diff
@@ -25,6 +25,19 @@ a {
   text-decoration: none;
   color: black;
```
use color variable for these colors.

Priya-Patel31 reviewed (approved):
Looks good and you can merge it .

tusharkandpal committed:
minor fixes

- added color variable
- changed border value
- changed margin padding of `start-section`

tusharkandpal committed:
changed shadow effect

ShreyashMahajan commented on utilities/global.css:
```diff
@@ -12,18 +12,33 @@
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 1000px;
```
use rem for more than 10px

tusharkandpal replied:
ya done thanks!

ShreyashMahajan reviewed (approved):
UI looks clean, Good to merge

tusharkandpal committed:
fix: px to rem
